content/es/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -0,0 +1,781 @@
+inicia correctamente,
+y luego pasando a "exitoso" (`Succeeded`) o "fallido" (`Failed`) si uno de los contenedores de un Pod
+termina en error.
+
+Mientras un Pod está en `Running`,
+el kubelet puede reiniciar sus contenedores para manejar algunos errores.
+Dentro de un Pod,
+Kubernetes rastrea diferentes [estados](#container-states) de contenedores y
+decide qué acción tomar para que el Pod esté sano otra vez.
+
+En la API de Kubernetes, los Pods tienen una especificación y un estado actual.
+El estado de un Pod consiste en un conjunto
+de [condiciones de un Pod](#pod-conditions).
+También puedes
+inyectar [información de estado personalizada](#pod-readiness-gate) en los datos
+de condiciones de un Pod, si es útil para tu aplicación.
+
+Los Pods se [programan](/docs/concepts/scheduling-eviction/) únicamente una vez
+en su tiempo de vida.
+Una vez que un Pod se programa (asigna) a un Nodo, el Pod se ejecuta en ese Nodo
+hasta que se termine o se [elimina](#pod-termination).
+
+<!-- body -->
+
+## Ciclo de vida de un Pod
+
+Igual que contenedores de aplicación individuales,
+se considera que los Pods son entidades relativamente efímeras
+(en lugar de durables).
+Los Pods se crean y se les 
+asigna un identificador único
+([UID](/es/docs/concepts/overview/working-with-objects/names/#uids)),
+y se programan para ejecutarse en nodos donde se mantienen hasta que se terminan
+(de acuerdo con las políticas de reinicio) o se eliminan.
+
+Si un {{< glossary_tooltip term_id="node" text="nodo" >}} muere,
+los Pods programados para ejecutarse en ese Nodo
+se [programan para eliminarse](#pod-garbage-collection) luego de un periodo de
+tiempo.
+
+Los Pods, por sí mismos, no se curan automáticamente.
+Si un Pod está programado para un {{< glossary_tooltip text="nodo" term_id="node" >}} y luego falla,
+el Pod se elimina; de la misma manera,
+un Pod no sobrevivirá a un desalojo debido a falta de recursos o mantenimiento
+del Nodo.
+Kubernetes utiliza una abstracción llamada {{< glossary_tooltip term_id="controller" text="controlador" >}}, que se encarga del trabajo de gestionar las instancias de Pod relativamente desechables.
+
+Un Pod determinado (según lo definido por un UID) nunca se "reprograma" a un nodo diferente; en cambio,
+ese Pod puede ser reemplazado por un Pod nuevo, casi idéntico, incluso con el mismo nombre si
+deseado, pero con un UID diferente.
+
+Cuando se dice que algo tiene la misma vida útil que un Pod, como un
+{{< glossary_tooltip term_id="volume" text="volumen" >}},
+eso significa que la cosa existe mientras ese Pod específico (con ese UID exacto)
+existe. Si ese Pod se elimina por cualquier motivo, e incluso si se requiere un reemplazo idéntico
+se crea, el objeto relacionado (un volumen, en este ejemplo) también se destruye y
+creado de nuevo.
+
+{{< figure src="/images/docs/pod.svg" title="Diagrama de un Pod" class=" diagram-medium" >}}
+
+Un Pod con múltiples contenedores que contiene un extractor de ficheros y un
```

Comment:
Add a missing asterisk to specify the note/reference to the above image (as the original content) :

```suggestion
*Un Pod con múltiples contenedores que contiene un extractor de ficheros y un
```